pkg/session_recording/recordings.jsx

```diff
@@ -529,7 +531,13 @@
         }
 
         componentDidMount() {
-            this.journalctlStart();
+            let proc = cockpit.spawn(["getent", "passwd", "tlog"]);
+
+            proc.stream((data) => {
+                this.uid = data.split(":",3)[2];
+                this.journalctlStart();
+            });
+
```

Comment:
This looks alright, with the exception of missing error handling, which might be good enough for the POC.
However, did you check with Cockpit API documentation and with Cockpit developers if there is a better way to do this than to invoke `getent` and parse its output?